packages/ui/core/src/app/modules/interfaces/interfaces.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class InterfacesService {
+  constructor(private http: HttpClient) {}
+
+  submitInterface(webhookUrl: string, request: FormData): Observable<any> {
```

Comment:
I believe we can make this typed instead of `any` following the same structure for interfaces -> return response.